tests/pytests/test_tdscf_excitations.py

```diff
@@ -24,1262 +23,218 @@
-            "e": 0.31362606792192205,
-            "sym": "A2"
-        }, {
-            "e": 0.34848280029565865,
-            "sym": "A1"
-        }, {
-            "e": 0.41301093564098723,
-            "sym": "B2"
-        }, {
-            "e": 0.4563016375242363,
-            "sym": "B2"
-        }, {
-            "e": 0.5822268659394665,
-            "sym": "A1"
-        }, {
-            "e": 0.7192503897441206,
-            "sym": "A2"
-        }, {
-            "e": 0.7532949620910153,
-            "sym": "B1"
-        }, {
-            "e": 0.8325669256159359,
-            "sym": "B2"
-        }, {
-            "e": 0.8418130551522903,
-            "sym": "A1"
-        }],
-        "RHF-3-wB97X-RPA-cc-pvdz": [{
-            "e": 0.21928320724732356,
-            "sym": "B1"
-        }, {
-            "e": 0.29090028689934033,
-            "sym": "A2"
-        }, {
-            "e": 0.29346538960855156,
-            "sym": "A1"
-        }, {
-            "e": 0.3566558496155522,
-            "sym": "B2"
-        }, {
-            "e": 0.4047901122018677,
-            "sym": "B2"
-        }, {
-            "e": 0.45354911327036923,
-            "sym": "A1"
-        }, {
-            "e": 0.7135101455725047,
-            "sym": "A2"
-        }, {
-            "e": 0.7332151322871041,
-            "sym": "B1"
-        }, {
-            "e": 0.7665614675068497,
-            "sym": "B2"
-        }, {
-            "e": 0.7836168280705303,
-            "sym": "A1"
-        }],
-        "RHF-3-wB97X-TDA-cc-pvdz": [{
-            "e": 0.22053635914394396,
-            "sym": "B1"
-        }, {
-            "e": 0.29207259028650134,
-            "sym": "A2"
-        }, {
-            "e": 0.2973130436723683,
-            "sym": "A1"
-        }, {
-            "e": 0.36116866641628476,
-            "sym": "B2"
-        }, {
-            "e": 0.4065136554262517,
-            "sym": "B2"
-        }, {
-            "e": 0.4588630653011993,
-            "sym": "A1"
-        }, {
-            "e": 0.7138849886618163,
-            "sym": "A2"
-        }, {
-            "e": 0.7338067963790569,
-            "sym": "B1"
-        }, {
-            "e": 0.7689722230616385,
-            "sym": "B2"
-        }, {
-            "e": 0.7875931047630326,
-            "sym": "A1"
-        }]
-    }
+# Reference data generated using G09.E01
+with open(Path(__file__).parent / "tdscf_reference_data.json") as f:
+    reference_data = json.load(f)
+
+# Canonical unrestricted system
+ch2 = psi4.geometry("""0 3
+C           0.000000    0.000000    0.159693
+H          -0.000000    0.895527   -0.479080
+H          -0.000000   -0.895527   -0.479080
+no_reorient
+no_com
+""",
+                    name="CH2")
+
+# Canonical restricted system
+h2o = psi4.geometry("""0 1
+O           0.000000    0.000000    0.135446
+H          -0.000000    0.866812   -0.541782
+H          -0.000000   -0.866812   -0.541782
+no_reorient
+no_com
+""",
+                    name="H2O")
```

Comment:
If any test has a `h2o` and optimizes it, I worry about the state of this local variable. May want to play it safe and fixture it.

```
@pytest.fixture
def mols():
    smols = {
        "H": """
0 2
H
""",
        "H2": """
1 2
H
H 1 1.0
""",
        "HeH": """
1 1
He
H 1 1.0
""",
    }

    return {k: psi4.core.Molecule.from_string(v) for k, v in smols.items()}
```

```
def test_weird_basis(subject, bas, ans, mols, request):
    """non-consecutive angular momentum basis set patterns"""

    mol = mols[subject]
```

Reply:
:+1: